cpp/src/slice2java/JavaUtil.h

```diff
@@ -237,11 +233,10 @@ namespace Slice
         // The functions return true if a custom type was defined and false to indicate
         // the default mapping was used.
         //
-        bool
-        getDictionaryTypes(const DictionaryPtr&, const std::string&, const MetadataList&, std::string&, std::string&)
-            const;
-        bool
-        getSequenceTypes(const SequencePtr&, const std::string&, const MetadataList&, std::string&, std::string&) const;
+        static bool
```

Comment:
I'm fine making these static but they probably don't need to be one `JavaGenerator`.

Reply:
I agree, but I didn't want to go too crazy in what was otherwise a very small and focused PR.